wporg-gp-translation-events.php

```diff
@@ -168,6 +192,7 @@ function submit_event_ajax() {
 	}
 	update_post_meta( $event_id, '_event_start_date', $start_date );
```

Comment:
AFAICT, the start and end datetimes are currently stored as values in the user's timezone. I think they should be stored in UTC instead, like discussed in https://github.com/WordPress/wporg-gp-translation-events/issues/4#issuecomment-1896466894

I'm imagining something along these lines (not tested :) ):

```php
$start_date = sanitize_text_field( $_POST['event_start_date'] );
$start_date = new DateTime( $start_date, new DateTimeZone( $event_timezone ) );
$start_date->setTimezone( DateTimeZone::UTC );
$start_date = $start_date->format( 'Y-m-d H:i:00' ); // The ':00' sets seconds to 0
```

Then, when displaying the event, we would need to convert from UTC to the event's timezone.